### ISSUE TITLE
point people in the beeminder forum to this

### ISSUE THREAD

**Issue by dreeves** [ssaadh/beeminder-basecamp#1]
Thanks so much for writing this! If you don't mind it getting a bit more attention could I persuade you to point people here from http://forum.beeminder.com?

**Comment by ssaadh** [comment]
Yep! I have to upload some code I didn't yet. The beeminder-harvest repo I have is much more finished and I was going to make it more non-tech friendly too, along with having it work with beeminder-toggl.

I don't personally use that, but "olimay" (not sure if his handle is the same on the forums for beeminder or how active he still is :P) was the one who directed me to apps like Beeminder and Complice. And he uses that. And since its free version is much more permissible vs Harvest's very limited free version, I was going to finish that first along with Harvest, with all the overlapping code.

I use Asana (the next beeminder repo I was going to publish and use) and Basecamp about equally these days. Both of those codebases will overlap quite a bit too. So I'll have to see the code I have locally for this beeminder-basecamp after Xmas and this weekend and see what I have left to do.
